### ISSUE TITLE
Site Audit: Check FAQ page for visual design consistency

### ISSUE THREAD

**Issue by NilakshiS** [hackforla/tdm-calculator#1959]

**Comment by namigoeku** [comment]
Hi @NilakshiS, could you clarify whether this includes both Admin-facing FAQ page as well?

**Comment by NilakshiS** [comment]
@namigoeku Yes, that was my intention. Admin FAQ page already includes all the features of user FAQ page, so we would not need another issue for user if cover just the admin-facing page.
